### PR TITLE
python3-s-tui: update to 1.1.4

### PR DESCRIPTION
**Pull request by pascal-huber**
#### Testing the changes
- I tested the changes in this PR: **briefly**

#### Local build testing
- I built this PR locally for my native architecture, x86_64-glibc
- I built this PR locally for these architectures:
  - x86_64-musl
  - aarch64 (crossbuild)
  - armv7l (crossbuild)
  -

### CODE REVIEW

File: srcpkgs/python3-s-tui/template [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'python3-s-tui'
 pkgname=python3-s-tui
-version=1.1.3
-revision=3
+version=1.1.4
+revision=1
 build_style=python3-module
 hostmakedepends="python3-setuptools"
 depends="python3-setuptools stress python3-urwid python3-psutil"
```
Comment (paper42) [inline srcpkgs/python3-s-tui/template:7]: what would you think about replacing stress with stress-ng?
```suggestion
depends="stress python3-urwid python3-psutil"
```

**Review by paper42 — COMMENTED** [review]
add python3-pytest to checkdepends

File: srcpkgs/python3-s-tui/template (hunk 1 above)
Comment (pascal-huber) [reply srcpkgs/python3-s-tui/template:7]: > what would you think about replacing stress with stress-ng?

Seams reasonable IMO. From what I can see `stress-ng` is (more) actively developed (and feature-rich). 

I changed it to `stress-ng`.

**Comment by pascal-huber** [comment]
> add python3-pytest to checkdepends

done.

**Pull request merged**